webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableConstants.js

```diff
@@ -13,9 +9,10 @@ import DropdownToggle from '../../../common/DropdownToggle';
 
 export const totalRiskFormatter = ({ title: totalRisk }) => ({
   children: (
-    <Section className="insights-total-risk" type="icon-group">
-      <InsightsLabel value={totalRisk} />
-    </Section>
+    <div>Hi</div>
+    // <Section className="insights-total-risk" type="icon-group">
```

Comment:
@jeremylenz Do we have a PR that removes our dependency on frontend-components? I suppose we will need to get it in before we can merge this one

Reply:
None that I know of as yet.

Reply:
Like https://github.com/theforeman/foreman_rh_cloud/pull/989 ? Or is that different

Reply:
Just to clarify, this is all the code we use: https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/InsightsLabel/InsightsLabel.tsx
https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/Section/Section.tsx
This is not a thi

Reply:
including imports and type definitions

Reply:
This should ve gotten resolved right after => https://github.com/theforeman/foreman_rh_cloud/pull/1001  ?